### ISSUE TITLE
WhitespaceAfter does not work on 'switch' and lambda statements

### ISSUE THREAD

**Issue by peterdemaeyer** [checkstyle/checkstyle#11298]
I want to be able to configure Checkstyle to report a violation on missing space after 'switch', which is currently not possible.
WhitespaceAfter detects a missing space after 'for', 'if' etc. but _not_ after 'switch'.
It would be nice if there would be a LITERAL_SWITCH similar to LITERAL_FOR etc.

https://checkstyle.org/config_whitespace.html#WhitespaceAfter
I have downloaded the latest cli from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

**How it works Now:**

```bash
peter@pangolin:~/Downloads/checkstyle$ javac Test.java 

peter@pangolin:~/Downloads/checkstyle$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
	"-//Puppy Crawl//DTD Check Configuration 1.3//EN"
	"http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">
<module name="Checker">
	<module name="TreeWalker">
		<module name="WhitespaceAfter"/>
	</module>
</module>

peter@pangolin:~/Downloads/checkstyle$ cat Test.java 
class Test {

	void test() {
		for(int i = 0; i < 10; i++) { // Checkstyle catches this, configurable with LITERAL_FOR.
		}
		switch(1) { // Checkstyle does NOT catch this, there is no LITERAL_SWITCH to configure.
			default:
				break;
		}
	}
}

peter@pangolin:~/Downloads/checkstyle$ RUN_LOCALE="-Duser.language=en -Duser.country=US"
peter@pangolin:~/Downloads/checkstyle$ java $RUN_LOCALE -jar checkstyle-9.3-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /home/peter/Downloads/checkstyle/Test.java:4:17: 'for' is not followed by whitespace. [WhitespaceAfter]
Audit done.
Checkstyle ends with 1 errors.
```

The problem is that WhitespaceAfter catches missing spaces after 'if', 'for' etc., but _not_ after 'switch'.
Similar to 'for', I suggest a configurable token LITERAL_SWITCH with the "token" property which is configured by default, and a similar output:

```bash
[ERROR] home/peter/Downloads/checkstyle/Test.java:6:21: 'switch' is not followed by whitespace. [WhitespaceAfter]
```

**Additional context**

The code style that I want to impose:

```java
for (int i = 0; i < 10; i++) {
}
switch (1) {
	default:
		break;
}
```

**Comment by romani** [comment]
Reasonable request to extend list of target tokens

**Comment by romani** [comment]
@Vyom-Yadav, can you help us with this issue ?

**Comment by Vyom-Yadav** [comment]
Sure, I am on it.
